test/parallel/test-tick-processor.js

```diff
@@ -35,7 +35,7 @@ if (common.isWindows ||
   console.log('1..0 # Skipped: C++ symbols are not mapped for this os.');
   return;
 }
-runTest(/RunInDebugContext/,
+runTest(/runInDebugContext/,
```

Comment:
Sorry, but this is not a **fix** for the problem. Something like https://github.com/nodejs/node/issues/5903#issuecomment-202126425 should be done to fix, I'm working with V8 authorities to resolve this.